crates/fuel-core/src/schema/balance.rs

```diff
@@ -50,6 +51,7 @@ pub struct BalanceQuery;
 
 #[Object]
 impl BalanceQuery {
+    #[graphql(complexity = "QUERY_COSTS.balance_query")]
```

Comment:
I'm confused where these `complexity` values are being parsed. I guess I'm not familiar enough with the `async_graphql::Object` api, but looking through the docs I can't find where you can extend it with custom fields like `complexity`.

Reply:
It's built into the proc-macro system: https://async-graphql.github.io/async-graphql/en/depth_and_complexity.html